### PR TITLE
HTCONDOR-2284 Add performance tuning of the AP to docs

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [ ] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [ ] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/admin-manual/ap-policy-configuration.rst [hunk 1]
```diff
@@ -571,3 +571,85 @@ See the :ref:`admin-manual/configuration-macros:condor_master configuration
...
+Performance Tuning of the AP
+----------------------------
+
+Of the three roles (AP, CM, EP) in a HTCondor system, the AP is the most common
+place performance tuning is done.  The CM is mostly stateless, and can
+typically scale out to very large pools without much additional work.  The EP
+daemons aren't resource intensive.  However, as the EP stores the state of all
+the jobs under its control, and persistently stores frequent updates to those
+jobs, it is not uncommon for the AP to exhaust system resources, like cpu, or
+disk and network bandwidth.
```
Comment (ColeBollig) [inline docs/admin-manual/ap-policy-configuration.rst:583]: Starts talking about EP and ends with AP... is this supposed to say `However, as the AP...`

File: docs/admin-manual/ap-policy-configuration.rst [hunk 2]
```diff
@@ -571,3 +571,85 @@ See the :ref:`admin-manual/configuration-macros:condor_master configuration
...
+
+Putting the schedd's database on the fastest disk
+'''''''''''''''''''''''''''''''''''''''''''''''''
+
+The *condor_schedd* frequently saves state to a file on disk, so that in event
+of a crash, no jobs will be lost on a restart.  The cost of this reliability,
+though, is relatively high.  In addition to writing to the disk, the schedd
+uses the fsync system call to force all the data onto the disk. By default,
+this file named job_queue.log is written to the :macro:`SPOOL` directory.
+However, if the :macro:`JOB_QUEUE_LOG` will override this path.  Setting
```
Comment (ColeBollig) [inline docs/admin-manual/ap-policy-configuration.rst:623]:
```suggestion
However, the configuration option :macro:`JOB_QUEUE_LOG` will override this path.  Setting
```

**Comment by ColeBollig** [comment]
Would this section also benefit from discussing/referencing DAGMan throttling and limits on scheduler/local universe jobs?

File: docs/admin-manual/ap-policy-configuration.rst (hunk 1 above)
Comment (GregThain) [reply docs/admin-manual/ap-policy-configuration.rst:583]: Oops. typo.  Fixed.

**Comment by GregThain** [comment]
> Would this section also benefit from discussing/referencing DAGMan throttling and limits on scheduler/local universe jobs?

Yes, but we can add that later.

**Pull request merged**